packages/edit-site/src/components/save-button/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp } from '@wordpress/core-data';
+import { useEffect, useState, useCallback } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { EntitiesSavedStates } from '@wordpress/editor';
```

Comment:
It seems like we're importing the editor package, just to use this component. What does this component do? I'm wondering whether it belongs to the editor package.

Reply:
It's the entities saving modal. Maybe it belongs in `core-data`?

Reply:
> It's the entities saving modal

What does it do exactly?

Reply:
>Isn't it the modal that lists entities with unsaved changes and lets the user choose if they want to update them all at once?

Yes

Reply:
Interesting, I guess`@editor` is the right place though but it makes me think we can reuse more than that from the Editor package (save buttons....) I also thought the components  from "editor" had a dependency on EditorProvider. I guess this one is a bit different?

Reply:
>but it makes me think we can reuse more than that from the Editor package (save buttons....)

Not without refactoring them.

>I also thought the components from "editor" had a dependency on EditorProvider. I guess this one is a bit different?

Yes

Reply:
Thinking more here, I feel this component might be misplaced.  We could be tempted to update it tomorrow to use the Editor store somehow to enable advanced flows in the editor screen and this will break the Edit Site screen without notice since it's not directly dependent on the editor store aka  EditorProvider.

Reply:
Can we move it to `core-data`?

Reply:
Maybe yes, I don't mind if it's done in a separate PR though to think it properly.